res/database/Default/user/scr0llbaer.xml

```diff
@@ -145,6 +145,19 @@
 			</description>
 			<data genre="4" price="1" quality="70" fictional="1" />
 		</news>
+
+		<news id="52328f89-ae30-49fc-add1-e455757220ef" type="0" creator="9551" comment="availability in thread id cujo-news-roter-platz was originally applied to the wrong (second) item, so the first item appeared at any date, but I found while play-testing the it still can be used in general and in addition when slightly modified to a fictional copycat event.">
+			<availability script="TIME_YEAR &gt;= 1988 &amp;&amp; TIME_SEASON=1" />
```

Comment:
>=1988

Reply:
The idea for this one is a fictional news about a copycat, so it can happen every year.

Reply:
The comment was messed up: please use `>=1988` rather than gt=...
One question would be, if this should be a "one-time"-Event (flags="2").

Reply:
done